src/test/java/api/items/ItemApiExamples.java

```diff
@@ -1383,6 +1421,208 @@ public void cannotRemoveHRID() throws Exception {
+      "status.name",
+      "Wrong name"
+    ));
+  }
+
+  @Test
+  public void cannotUpdateItemWithWrongStatus() throws Exception {
+    JsonObject createdInstance = createInstance(smallAngryPlanet(UUID.randomUUID()));
+
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    IndividualResource postResponse = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingId)
+      .temporarilyInReadingRoom());
+
+    JsonObject createdItem = postResponse.getJson();
+    assertThat(createdItem.getJsonObject("status").getString("name"),
+      is(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject updatedItem = createdItem.copy()
+      .put("status", new JsonObject().put("name", "Wrong name"));
+
+    Response updateResponse = updateItem(updatedItem);
+    assertThat(updateResponse, hasValidationError(
+      "Undefined status specified",
+      "status.name",
+      "Wrong name"
+    ));
+  }
+
+  @Test
+  public void cannotRemoveEntireStatus() throws Exception {
+    JsonObject createdInstance = createInstance(smallAngryPlanet(UUID.randomUUID()));
+
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    IndividualResource postResponse = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingId)
+      .temporarilyInReadingRoom());
+
+    JsonObject createdItem = postResponse.getJson();
+    assertThat(createdItem.getJsonObject("status").getString("name"),
+      is(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject updatedItem = createdItem.copy();
+    updatedItem.remove("status");
+
+    Response updateResponse = updateItem(updatedItem);
+    assertThat(updateResponse,
+      hasValidationError("Status is a required field", "status", null)
+    );
+  }
+
+  @Test
+  public void canRemoveStatusName() throws Exception {
+    JsonObject createdInstance = createInstance(smallAngryPlanet(UUID.randomUUID()));
+
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    IndividualResource postResponse = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingId)
+      .temporarilyInReadingRoom());
+
+    JsonObject createdItem = postResponse.getJson();
+    assertThat(createdItem.getJsonObject("status").getString("name"),
+      is(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject updatedItem = createdItem.copy()
+      .put("status", new JsonObject());
+
+    Response updateResponse = updateItem(updatedItem);
+    assertThat(updateResponse,
+      hasValidationError("Status is a required field", "status", null)
+    );
+  }
+
+  @Test
+  public void canSetCopyNumberProperty() throws Exception {
```

Comment:
Similarly to the storage module, are there tests for general creating or updating an item?

Reply:
This test and `canUpdateCopyNumberProperty` have additional statements to verify that old `copyNumbers` property is not present in item-storage (see line # 1552).

I think it make sense to have them separately.

Reply:
Ok. As API tests are expensive to maintain, when should this separate test be removed?

Reply:
hmm... You're right, I think thees checks only important for refactoring step. Will remove them.